### DISCUSSION TITLE
Which inputs are configured for ext interrupts?

### DISCUSSION THREAD

**Discussion by dresco** [grblHAL/STM32F7xx#4] [Q&A] [answered]
Hi, was just looking at the reference map file, and noticed that (for instance) X_LIMIT_PIN and AUXINPUT0_PIN are both mapped to pin #15. 

This got me wondering, which inputs need an external interrupt (and therefore a dedicated pin number)? Is this documented somewhere?  Thanks..

**Comment by terjeio** [comment, marked as answer]
Limit and control inputs should always be on interrupt capable pins, probe pin also if possible (can be used for continus probe protection when I get around to implement that). Auxillary inputs can have more uses if on interrupt capable pins but is not a neccessity - there is a discovery mechanism for plugin code to search for pins with the required properties.

**Reply by dresco** [reply]
Thanks, just starting to think about layout for an H7 breakout..
